README.md

```diff
@@ -34,6 +34,18 @@ for help in case of questions.
 
 Documentation for developers can be found at [DEVELOPING.md](DEVELOPING.md).
 
+## Development setup
+This app requires [Text](https://github.com/nextcloud/text) and [Circles](https://github.com/nextcloud/circles).
+You also need nvm v16.16.0 to compile the JavaScript. 
```

Comment:
You likely mean node v16 which can be installed using nvm ;)

Reply:
Thank you Marcel!! Added it to the file =)